paper_sections/02-gams.Rmd

```diff
@@ -115,17 +134,17 @@ basis, and comes into play with tensor products...**
 **Do we actually need the cubic spline here? It would simplify the presentation to 
 just show the thin plate spline, and it gets the idea across.**
 
-![\label{fig:basis_example}a) Examples of the basis functions associated with a six basis function thin plate spline (top) and a cubic spline (bottom) calculated for x data spread evenly between $x=0$ and $x=1$. Each line represents a single basis function. To generate a given smooth function, each basis function would be multiplied by its own coefficient. b) The smoothing and shinkage penalty matrices for the thin plate and cubic smoothers shown on left. Red entries indicate positive values and blue indicate negative values. For example, for the thin plate spline, functions f3 and f4 would have the greatest proportionate effect on the total penalty (as they have the largest values on the diagonal), whereas function f6 would not contribute to the penalty at all (all the values in the 6th row and column of
-the penalty matrix are zero). This means function f6 is in the null space of this basis,
-and would be treated as completely smooth.](../figures/example spline basis and penalties.png)
+**GLS: we've glossed over exactly how TPRS bases are created, the low-rank version of Wood, nothing on knots...**
```

Comment:
if we can avoid talking about knots I think we should, as it overcomplicates things imo, especially when explaining tprs... but I am happy to be overruled by a good explanation :)

Reply:
So we're not going to talk about the CRS basis then? [There's a note from @dill to put it back]. And if we don't talk about CRS then should we be including them in this figure with the basis functions and penalty matrices?

Reply:
I think we don't need to, besides just mentioning that there are other smoothers, some of which might be faster for big data, etc.  We should mention cyclic cubic smoothers, as we do use them in the paper, but that can just be a note in this section.

I put together a version of this figure with just the thin plate spline and its penalty matrix (I got rid of the null space penalty matrix, as it only really makes sense in specific contexts, and is really too complicated for this paper). It's on my work computer, but I'll post it tomorrow.

Reply:
Hi all,

Here's the figure with just thin plate splines, and omitting the null space. I also included the intercept term (before I absorbed that, so that I was effectively plotting a k=7 smoother).

![example spline basis and penalties](https://user-images.githubusercontent.com/1554249/41109497-ba3923ac-6a51-11e8-98bd-cfb7e7d910b9.png)

Reply:
I like this! 😻